src/lib/controllers/ui-controller.js

```diff
@@ -237,7 +240,20 @@ export default class UIController {
       enableLemmaTranslations: false,
       irregularBaseFontSizeClassName: 'alpheios-irregular-base-font-size',
       // Whether to disable text selection on mobile devices
-      disableTextSelection: false
+      disableTextSelection: false,
+      /*
+      textLangCode is a language of a text that is set by the host app during a creation of a UI controller.
+      It has a higher priority than a `preferredLanguage` (a language that is set as default on
+      the UI settings page). However, textLangCode has a lower priority than the language
+      set by the surrounding context of the word on the HTML page (i.e. the language that is set
+      for the word's HTML element or for its parent HTML elements).
+      The value of the textLangCode must be in an ISO 639-3 format.
+      A host application may not necessarily set the current language. In that case
+      it's value (which will be null by default) will be ignored.
+       */
+      textLangCode: null,
+      // If set to true, will use the `textLangCode` over the `preferredLanguage`
+      useTextLang: false
```

Comment:
I have only one small notice - only after reading all the code changes and read the last comment I understood that
useTextLang means useR text language, but not used before

if it is not difficult and you decide that it is worth to do - may be it would be more meanful to use here `userTextLang` instead of `useTextLang` ?

Reply:
That isn't quite right, I don't think. The meaning of 'useTextLang' here is to use the language set by the provider of the library to the page.  The **user** might override that decision by actively changing the preferredLanguage setting.   So, maybe actually 'overridePreferredLanguage' would be clearer?  What do you think @kirlat ?

Reply:
What I've meant with the `useTextLang` name is that we have a `textLang` code prop that, when set, will have priority over `preferredLanguage`. So if `textLang` is present, `useTextLang` is set to true to indicate that we have a valid `textLang` value and it will pervail over the `preferredLanguage` (so we're "using" a `textLang`). But once the user changes the `preferredLanguage` value, `preferredLanguage` starts to take priority over the `textLang` and `textLang` is not important any more (so we're not "using" it).
So the meaning I was putting into `useTextLang` was whether to use (or not to use) `textLang` over `preferredLanguage`. I guess `userTextLang` would also makes sense in this context. Not sure what name is clear. Maybe something else? Would really love to avoid any confusion here. @balmas, what do you think?

Reply:
> So, maybe actually 'overridePreferredLanguage' would be clearer? 

I think this is a good idea. Will change to that